x/pairing/types/scores/score_req.go

```diff
@@ -0,0 +1,76 @@
+package score
```

Comment:
this file is more than the ScoreReq, either change its name or move the other structs to another file

Reply:
done. see https://github.com/lavanet/lava/pull/594/commits/c1af94249ab68d6734ff129f2d937f389db2fba2

Reply:
disregard last comment (moving files again). Will update commit soon

Reply:
done. see https://github.com/lavanet/lava/pull/594/commits/0372ac47a9f29e94684b16557db8de6e693ffc9c